scarb/src/bin/scarb/interactive.rs

```diff
@@ -13,9 +21,13 @@ pub fn ask_for_test_runner() -> Result<TestRunner> {
         ]
     };
 
-    let answer = Select::new("Which test runner do you want to set up?", options).prompt()?;
+    let selection = Select::with_theme(&ColorfulTheme::default())
```

Comment:
how is this working when colors are turned off? (Scarb UI is providing this information)

Reply:
When tested on `export TERM=dumb` it works normally (of course without colors).

Reply:
and what about these envs https://docs.rs/console/latest/console/fn.colors_enabled.html?

Reply:
As expected, there is no color, but nothing breaks.

Reply:
perfect!